nipype/interfaces/base/core.py

```diff
@@ -166,9 +166,15 @@ class BaseInterface(Interface):
     _redirect_x = False
     references_ = []
     resource_monitor = True  # Enabled for this interface IFF enabled in the config
+    check_version = None
```

Comment:
Might choose a different name, as `version` attached to an `Interface` feels like it should apply to the interface itself, not nipype. Maybe:

```suggestion
    _etelemetry_version_data = None  # Dictionary to be populated on first interface init
```